assignment4.Rmd

```diff
@@ -111,8 +144,49 @@ biplot(pca)
 
 #Calculate values for each student that represent these your composite variables and then create a new correlogram showing their relationship to mean_correct.
 
+
```

Comment:
Could think about how to calculate this.

Reply:
PCs are linear composites of the original variables and it is like describing vectors in a different n-dimensional coordinate.